src/mescal/tapis_de_v3/jobs.clj

```diff
@@ -78,37 +108,39 @@
   (->> (assoc (prepare-params tapis app (:paramPrefix submission) (:config submission))
               :name              (build-job-name submission)
               :appId             (:app_id submission)
-              :archive           true
+              :appVersion        (:version app)
               :archiveOnAppError true
-              :archivePath       (.tapisFilePath tapis (:output_dir submission))
-              :archiveSystem     (.storageSystem tapis)
-              :notifications     (job-notifications (:callbackUrl submission)))
+              :archiveSystemDir  (.tapisFilePath tapis (:output_dir submission))
+              :archiveSystemId   (.storageSystem tapis)
+              :subscriptions     (job-notifications (:callbackUrl submission))
+              :notes             {:appName        (app-listings/get-app-name app)
+                                  :appDescription (app-listings/get-app-description app)})
```

Comment:
We can add any notes we want in a job submission.
These fields are used in the `format-job-submisison-response` function.